modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc

```diff
@@ -377,3 +388,117 @@ Scanner output rules are used in daemon/socket mode.
+----
+
+- Enable the app as admin in ownCloud under Settings > Apps or with the following occ command:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} app:enable files_antivirus
+----
+
+
+=== General Configuration
+
+ICAP ist configured via occ commands. Go to your ownCloud directory, usually `/var/www/owncloud`, and enter the following occ commands:
+
+* To set the IP address of your anti-virus server:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set files_antivirus av_host --value="172.17.0.3"
+----
+
+* Specify the port of the anti-virus server:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set files_antivirus av_port --value="1344"
+----
+
+* Set the mode to ICAP:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set files_antivirus av_mode --value="icap"
+----
+
+The setting 'icap' triggers a grace period of 24 hours if you don't have an Enterprise license but want to test ICAP.
+
+* Specify what do do with the offending file:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set files_antivirus av_infected_action --value="delete"
+----
+
+Possible values are 'delete' and 'only_log'.
+
+
+=== Run with c-icap/ClamAV
+
+c-icap has a built in ClamAV module. For more information see: {c-icap-url}[c-icap on sourceforge]
+
+An out of the box docker image is available {docker-url}[here].
+
+For simple local testing, run `docker run -ti deepdiver/icap-clamav-service` and get its IP address using docker inspect.
+
+The IP address needs to be set in the general configuration above. Then set the mode to 'c-icap with ClamAV':
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} occ config:app:set files_antivirus av_request_service --value="avscan"
+----
+
+Then set the response header:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set files_antivirus av_response_header --value="X-Infection-Found"
+----
+
+
+### Run with Kaspersky Anti-Virus (KAV)
+
+Kaspersky provides {kav-docker-url}[docker images] as well.
+Follow the instructions in Kaspersky ScanEngine for Kubernetes.7z.
+
+Additional configuration: Set <Configuration><ICAPSettings><Allow204>1</Allow204></ICAPSettings></Configuration> in kavicapd.xml
+
+Set KAV as virus scanner with the value 'req':
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set files_antivirus av_request_service --value="req"
+----
+
+And the respective response header:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set files_antivirus av_response_header --value="X-Virus-ID"
+----
+
+NOTE: The older versions of KAV did not send back the virus/infection name in an ICAP header.
+
+In v2.0.0 the header to transport the virus can be configured. Default: No header is sent.
+
+```
```

Comment:
```
[source,console]
----
sed ...
----
```
What is ths command doing, from where should this command be run, do you need previleges (sudo)?

Reply:
I'd just drop that - one should not use sed to maniplate xml ..... furthermore: people need to edit their config in what ever way fits their envrionment.

Reply:
Done.